ml-agents/README.md

```diff
@@ -24,19 +41,18 @@ agent must either use an **External** brain or use a brain that is broadcasting
 actions for agents with external brains, but can only observe broadcasting
 brains (the information you receive for an agent is the same in both cases). See
 [Using the Broadcast
-Feature](Learning-Environment-Design-Brains.md#using-the-broadcast-feature).
+Feature](../docs/Learning-Environment-Design-Brains.md#using-the-broadcast-feature).
```

Comment:
The problem with this relative path is that on the PyPi website itself, the `docs` directory won't be there. Maybe we should put a link to the file on GitHub?